packages/application-shell/src/components/authenticated/authenticated.spec.js

```diff
@@ -90,7 +93,17 @@ describe('rendering', () => {
       it('should not put `isAuthenticated` into local storage', async () => {
         await wait(
           () => {
-            expect(storage.put).not.toHaveBeenCalledWith();
+            expect(storage.put).not.toHaveBeenCalled();
```

Comment:
This fella was a false positive (passing falsely) as the mock was not called with `undefined`.